algosdk/future/transaction.py

```diff
@@ -3008,3 +3008,32 @@ def assign_group_id(txns, address=None):
             tx.group = gid
             result.append(tx)
     return result
+
+def wait_for_confirmation(algod_client, transaction_id, timeout=None, **kwargs):
```

Comment:
```suggestion
def wait_for_confirmation(algod_client, txid, wait_rounds=1, **kwargs):
```
Rationale:
txid is most used ident for txn ids across all the code
timeout usually assumes time, but the func accepts rounds

default value needs to be meaningful since the func is supposed to wait at least round.
So I suggest to set `current_round = last_round + wait_rounds` below